package.json

```diff
@@ -122,9 +122,9 @@
     "chance": "^1.1.8",
     "css-loader": "^6.8.1",
     "cypress": "^9.1.0",
-    "eslint": "^8.49.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-cypress": "^2.10.3",
+    "eslint": "8.46.0",
```

Comment:
`eslint` version downgrade?

Reply:
I've done the upgrade manually to avoir regression
https://github.com/Qovery/console/pull/921/commits/1b60c11403b08e427ffa6a6050fb91b57eacbb22